.eslintrc

```diff
@@ -68,7 +68,12 @@
         "ignoreParameters": true
       }
     ],
-    "@typescript-eslint/no-unused-vars": "warn",
+    "@typescript-eslint/no-unused-vars": [
+      "warn",
+      {
+        "argsIgnorePattern": "^_"
```

Comment:
So we can prefix unused args with underscores and not have to add eslint-ignore. It'll make the code easier for read because some instances of eslint-ignore don't include any context about what's being ignored.